test-e2e/common/command-env.test.js

```diff
@@ -0,0 +1,30 @@
+// @flow
+
+const path = require('path');
+const fs = require('fs-extra');
+
+const {initFixture, promiseExec} = require('../test/helpers');
+
+describe('Common - command-env', () => {
+  it('generates valid environmenmt with deps and devdeps in $PATH', async () => {
+    expect.assertions(2);
+    const p = await initFixture(path.join(__dirname, 'fixtures/simple-project'));
+    await p.esy('build');
+
+    const commandEnv = (await p.esy('command-env')).stdout;
+
+    await fs.writeFile(path.join(p.projectPath, 'command-env'), commandEnv);
+
+    await expect(
+      promiseExec('source ./command-env && dep', {
```

Comment:
Things like this likely won't work on Windows - this hits the same blocker as #272 - but that's OK, we can inventory these and tackle them as a next step.

Reply:
As you suggested we can add something like this https://github.com/facebook/jest/blob/84f0ea307e65c1d8a530781aeab06da4cd5a97a1/scripts/ConditionalTest.js#L33. I think we should do that in a separate PR. On the other hand most (all?) the tests that won't run on windows are in this part so it could make sense to add it here.